src/redux/books/bookSlice.jsx

```diff
@@ -44,15 +44,13 @@ export const addBook = createAsyncThunk(
 
 export const deleteBook = createAsyncThunk('books/deleteBook', async (item_id, api) => {
     try {
-        const res = await axios.delete(`${deleteUrl}/${item_id}`);
-        return res.data
+        const response = await axios.delete(`${deleteUrl}/${item_id}`);
+        return response.data
     } catch (error) {
         return api.rejectWithValue('error connecting to api')
     }
 })
```

Comment:
- [ ] Following the previous reviewer's request, could you please move your reducers inside the slice as actions? Just a friendly reminder that this is a project requirement. 

   ![image](https://github.com/Simpleshaikh1/book-store-cra/assets/107952724/8bba4a26-d965-4447-9216-8ccaefb4a6fe)

Reply:
As suggested by the previous reviewer, kindly add your reducers inside the bookSlice as this is part of the requirement,You didn't make any changes the previous reviewer suggested. This is an example of a snippet code you can look up :


 ` import { createSlice } from "@reduxjs/toolkit"

export const booksSlice = createSlice({
  name: "books",
  initialState: {
    books: [],
  },
  reducers: {
    addBook: (state, action) => {
      state.books = [
        ...state.books,
        {
          id: Math.floor(Math.random() * 100),
          title: action.payload.title,
          author: action.payload.author,
        },
      ]
    },
    removeBook: (state, action) => {
      state.books = state.books.filter((book) => book.id !== action.id)
    },
  },
})

export const { addBook, removeBook } = booksSlice.actions

export default booksSlice.reducer
`